pvlib/iotools/pvgis.py

```diff
@@ -516,17 +522,16 @@ def get_pvgis_tmy(latitude, longitude, outputformat='json', usehorizon=True,
     elif outputformat == 'csv':
         with io.BytesIO(res.content) as src:
             data, months_selected, inputs, meta = _parse_pvgis_tmy_csv(src)
-    elif outputformat == 'basic':
-        with io.BytesIO(res.content) as src:
-            data, months_selected, inputs, meta = _parse_pvgis_tmy_basic(src)
     elif outputformat == 'epw':
         with io.StringIO(res.content.decode('utf-8')) as src:
             data, meta = parse_epw(src)
             months_selected, inputs = None, None
+    # raise exception if pvgis format isn't in ['csv', 'json', 'epw']
```

Comment:
unnecessary, this line will never be reached because pvgis will return 400 BAD REQUEST if the `outputformat` isn't one of the options specified by the API